ui/tests/e2e/test.js

```diff
@@ -11,219 +11,155 @@ describe("End-to-end", () => {
   beforeAll(async () => {
     // It can take a while for servers to start up
     jest.setTimeout(JEST_TIMEOUT_MS);
+    jest.setTimeout(10000000);
```

Comment:
Is this necessary? Why?

Reply:
That was there for debugging, good catch. Deleted.